src/components/BuildArtifactDependencyGraphPage/BuildArtifactDependencyGraphPage.tsx

```diff
@@ -170,7 +171,7 @@ export const BuildArtifactDependencyGraphPage = ({ componentId = 'a1' }: IBuildA
           <ServiceContainerLoading {...serviceContainerDependencyGraph} hasSkeleton title="Build Artifact Dependency Graph">
             <BuildArtifactDependencyGraph
               data={serviceContainerDependencyGraph.data}
-              mainNode={serviceContainerBuild.data?.id}
+              mainNode={serviceContainerBuild.data!.id}
```

Comment:
I am not familiar with this component, but as far as I know, this gets results from the endpoint `/v2/builds/{id}`, and it either returns data or just a 404/500 response. since we covered the 404 and 500 status, this code should be fine.